src/shared_realm.hpp

```diff
@@ -210,6 +212,7 @@ class Realm : public std::enable_shared_from_this<Realm> {
     void invalidate();
     bool compact();
     void write_copy(StringData path, BinaryData encryption_key);
+    BinaryData write_copy_to_mem();
```

Comment:
There's no way for a caller to know that they are required to free the memory this returns. `OwnedBinaryData` would be better.